client/src/pages/TrackEditPage/TrackEditPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+import Card, { CardBody } from '../../components/Card';
+import { connect } from 'react-redux';
+import FormTrackEdit from '../../components/forms/FormTrackEdit';
+
+class TrackEditPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const { match, track } = this.props;
+    const trackId = match.params.id;
+    const {
+      name,
+      track_number,
+      duration_in_seconds,
+      disk_number,
+      album,
+    } = track.doc;
+    return (
+      <div className="track-edit-page">
+        <Card>
+          <CardBody>
+            <h1>Edit Track</h1>
+            <FormTrackEdit
```

Comment:
Rather than passing the track data in as properties, can you change the `FormTrackEdit` component so it gets the track data from Redux?